package/MDAnalysis/coordinates/base.py

```diff
@@ -1228,13 +1228,22 @@ def __init__(self, filenames, **kwargs):
 
         .. versionchanged:: 0.8
            The *delta* keyword was added.
+        .. versionchanged:: 0.13
+           The *delta* keyword was changed to *frame_dt*.
         """
         self.filenames = asiterable(filenames)
-        self.readers = [core.reader(filename, **kwargs) for filename in self.filenames]
-        self.__active_reader_index = 0  # pointer to "active" trajectory index into self.readers
+        self.readers = [core.reader(filename, **kwargs)
+                        for filename in self.filenames]
+        # pointer to "active" trajectory index into self.readers
+        self.__active_reader_index = 0
 
         self.skip = kwargs.get('skip', 1)
-        self._default_delta = kwargs.pop('delta', None)
+        if 'delta' in kwargs:
+            warnings.warn("Keyword 'delta' is now deprecated "
+                          "(from version 0.13); "
+                          "use 'frame_dt' instead", DeprecationWarning)
+        # Give priority to 'frame_delta' over the deprecated 'delta'
+        self._default_delta = kwargs.pop('frame_dt', kwargs.pop('delta', None))
```

Comment:
I'd also prefer not to introduce yet another variation of delta/dt to keep the interface reasonably consistent.